routing/single_mwm_router.cpp

```diff
@@ -106,6 +106,8 @@ IRouter::ResultCode SingleMwmRouter::DoCalculateRoute(MwmSet::MwmId const & mwmI
   RoadPoint const start(startEdge.GetFeatureId().m_index, startEdge.GetSegId());
   RoadPoint const finish(finishEdge.GetFeatureId().m_index, finishEdge.GetSegId());
 
+  EstimatorGuard guard(mwmId, *m_estimator);
```

Comment:
EstimatorGuard is used only here. Please, move it to the anonymous namespace in this cpp file.